lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -781,8 +784,12 @@ export abstract class ClientApplication {
             const authClient = await this.createAuthCodeClient(serverTelemetryManager, requestAuthority, validRequest.correlationId);
             browserRequestLogger.verbose("Auth code client created");
 
-            // create logout string and navigate user window to logout. Auth module will clear cache.
+            // Create logout string and navigate user window to logout.
             const logoutUri: string = authClient.getLogoutUri(validRequest);
+
+            // Clear cache on logout
+            await authClient.clearCacheOnLogout(validRequest);
```

Comment:
What should happen if this throws?

Reply:
The current implementation doesn't do error handling for the cache clearing, open to suggestions on how we should proceed if this part throws and the cache isn't cleared successfully. Is try/catch and logging the failure sufficient?